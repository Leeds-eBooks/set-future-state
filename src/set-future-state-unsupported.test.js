// @flow

describe('When WeakMap is not available', () => {
  it('throws', () => {
    // eslint-disable-next-line no-global-assign, no-native-reassign
    WeakMap = void 0
    expect(() => require('./set-future-state')).toThrowErrorMatchingSnapshot()
  })
})
