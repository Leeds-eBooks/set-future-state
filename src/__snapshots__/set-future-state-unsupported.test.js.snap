// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`When WeakMap is not available throws 1`] = `"\`setFutureState\` requires WeakMap to be available. Consider including a polyfill such as core-js."`;
